openpype/hosts/traypublisher/plugins/publish/collect_shot_instances.py

```diff
@@ -120,7 +120,6 @@ def _solve_inputs_to_data(self, instance):
         frame_dur = frame_end - frame_start
 
         return {
-            "asset": _cr_attrs["asset_name"],
```

Comment:
Won't this break OpenPype editorial?

Reply:
it should not, since the asset in data of instance is already defined in creators, this was removal of redundancy.
look to [`openpype\hosts\traypublisher\plugins\create\create_editorial.py:704`](https://github.com/ynput/OpenPype/pull/5873/files#diff-8a769093646ad52ee90485fdaad694db18055ac47f411f70fe22ab36af5b5056L700)